lib/daru.rb

```diff
@@ -95,26 +95,20 @@ def plotting_library= lib
 require 'forwardable'
 require 'erb'
 require 'date'
-
```

Comment:
No specific reason - but as they were already grouped properly, I thought that placeholders weren't necessary. I've revived these lines in the last commit. :smile: